components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -295,25 +300,9 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
         this.notificationVisible = isError;
       });
 
-    this.isWorkflowEnabled$ = this.store.select(createSelector(clientRunsState,
-      (state) => state.workflowEnabled));
-
     this.columns$ = this.store.select(clientRunsColumns);
 
-    combineLatest(
-      this.store.select(createSelector(
-        state => state.sidebar, (state) => state.selectedOrgs)),
-      this.store.select(createSelector(
-        state => state.sidebar, (state) => state.selectedChefServers)),
-      this.store.select(createSelector(
-        clientRunsState, (state) => state.nodeFilter)))
-      .pipe(
-        map(([selectedOrgs, selectedChefServers, nodeFilter]) => {
-          nodeFilter.organizations = selectedOrgs;
-          nodeFilter.servers = selectedChefServers;
-          return nodeFilter;
-        }))
-      .subscribe(
```

Comment:
All this confusing logic is not needed because all the filter parameters are now in the search bar.